### PR TITLE
Update quay operator version

### PR DESCRIPTION
**Pull request by kmendez-redhat**
##### SUMMARY
Updated the security lab workload so that it modifies the version of the quay operator from latest to v0.0.1, matching the version that is expected for deployment

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
ocp4-workload-security-compliance-lab

##### ADDITIONAL INFORMATION
The latest version of the operator is used to deploy a later version of quay than what is currently expected by the lab.

### CODE REVIEW

File: ansible/roles/ocp4-workload-security-compliance-lab/tasks/quay.yml [hunk 1]
```diff
@@ -9,6 +9,12 @@
     dest: "{{ tmp_dir }}/quay-operator"
     version: "4ba1d1620e92776dd40decad9e8d2cc5ac49df7f"
 
+- name: Update operator version
+  replace:
+    path: "{{ tmp_dir }}/quay-operator/deploy/operator.yaml"
+    regexp: "image: quay.io/redhat-cop/quay-operator:latest"
+    replace: "image: quay.io/redhat-cop/quay-operator:v0.0.1"
```
Comment (fridim) [inline ansible/roles/ocp4-workload-security-compliance-lab/tasks/quay.yml]: Maybe we can ask Andrew to add a parameter to overwrite `latest` ?

**Comment by fridim** [comment]
can you rebase from development, the CI was broken ?

**Comment by kmendez-redhat** [comment]
@fridim rebased - looks like the ci is passing now, thanks!

**Comment by fridim** [comment]
What do you think of:
> Maybe we can ask Andrew to add a parameter to overwrite latest ?

Do you want to get in touch with Andrew first ?

File: ansible/roles/ocp4-workload-security-compliance-lab/tasks/quay.yml (hunk 1 above)
Comment (kmendez-redhat) [reply ansible/roles/ocp4-workload-security-compliance-lab/tasks/quay.yml]: @fridim right now this is pretty short turnaround - we're actually checking out Andrew's git repo and switching to a specific commit (pre-summit). Moving forward, we can probably work with him on maybe creating a new branch from this commit to support the older version of Quay with a dynamic version (or just updated to point to this version)

**Pull request merged**